ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -247,7 +247,7 @@ public void start() throws StoreException {
             diskIOScheduler, metrics, time, sessionId, storeDescriptor.getIncarnationId());
         compactor.initialize(index);
         metrics.initializeIndexGauges(storeId, index, capacityInBytes);
-        long logSegmentForecastOffsetMs = TimeUnit.DAYS.toMillis(config.storeDeletedMessageRetentionDays);
+        long logSegmentForecastOffsetMs = TimeUnit.HOURS.toMillis(config.storeDeletedMessageRetentionHours);
         long bucketSpanInMs = TimeUnit.MINUTES.toMillis(config.storeStatsBucketSpanInMinutes);
```

Comment:
@justinlin-linkedin if `logSegmentForecastOffsetMs` becomes a few hours and `bucketSpanInMs`, `bucketCount` stay unchanged, will it be a problem? `logSegmentForecastEndTimeMsForDeleted`  > scan timestamp

Reply:
@justinlin-linkedin  any comment on Ying's concern?

Reply:
@jsjtzyy @zzmao  I don't think so, buckets are designed to serve the incoming requests, it shouldn't be affected by the retention duration. One we get a smooth transition is to not enable bucket reporting in delivery cluster.

Reply:
Thanks and noted. I will disable bucket reporting in EI testing.

Reply:
Sure, we can temporarily disable bucket and revisit this before turning it on.